python/pyspark/sql/tests.py

```diff
@@ -5925,6 +5925,22 @@ def test_invalid_args(self):
                     'mixture.*aggregate function.*group aggregate pandas UDF'):
                 df.groupby(df.id).agg(mean_udf(df.v), mean(df.v)).collect()
 
+    def test_self_join_with_pandas(self):
```

Comment:
Just realized this test is in a wrong class. This should be moved to GroupedMapPandasUDFTests